scripts/gulp/tasks/universal.ts

```diff
@@ -1,28 +0,0 @@
-/**
- * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/LICENSE
- */
-
-import { series, task } from 'gulp';
-
-import { generate } from '../../prerender/ngsw-config';
-import { generateSitemap } from '../../prerender/sitemap';
-import { execNodeTask } from '../util/task-helpers';
-
-/** Run prerender script on the output directory, to render static html. */
-task(':run:prerender', execNodeTask('@angular/cli', 'ng', ['run', 'ng-zorro-antd-doc:prerender']));
```

Comment:
This task has been removed and we no longer need a separate pre-rendering build, as pre-rendering is already supported by ng build using the application builder.